apps/worker/lib/worker/docker/pipeline.ex

```diff
@@ -3,14 +3,12 @@ defmodule Worker.Docker.Pipeline do
   require Logger
 
   def start_link(producer) do
-    Flow.from_stages([producer], stages: 1, max_demand: 10)
+    Flow.from_stages([producer], stages: 1, max_demand: 5)
```

Comment:
Does it make sense to have this configurable through an env var as well?

Reply:
I can make that change